concepts/05 UI Components/DataGrid/00 Getting Started with DataGrid/80 Export Data to Excel.md

```diff
@@ -207,6 +207,8 @@ This method can be called at any point in your application. The following code c
 
 Run the code and click the Export button in the upper-right corner. Grid data should be exported to the DataGrid.xlsx file.
 
+[note]You can also export DataGrid to CSV. To do this, implement the **exportDataGrid(options)** method in the same way as shown in the following ticket: <a href="https://supportcenter.devexpress.com/ticket/details/t920593/pivotgrid-exceljs-export-to-export-pivotgrid-into-csv-file" target="_blank">Export PivotGrid into CSV file</a>.
```

Comment:
I see this note is repeated in three places. How about an include?

Reply:
@RomanTsukanov There are two notes and one edited comment to be precise,